tests/bridge.py

```diff
@@ -7,21 +7,25 @@
 class TestBridge(unittest.TestCase):
 
     def setUp(self):
-        self.link = bench.links[0]
-        self.link.host_if.flushAddresses()
-        self.link.host_if.addAddress("192.168.10.2/24")
+        links = bench.links
+
+        for i, l in enumerate(links, start=1):
```

Comment:
`for l in links:` is enough here, `enumerate` is useful only if we want to get the index as well.